blindsign/blindrsa/brsa_test.go

```diff
@@ -503,41 +558,53 @@ func Example_blindrsa() {
 	// Server: generate an RSA keypair.
 	sk, err := rsa.GenerateKey(rand.Reader, 2048)
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "failed to generate RSA key: %v", err)
+		fmt.Printf("failed to generate RSA key: %v", err)
 		return
 	}
 	pk := &sk.PublicKey
 	server := NewSigner(sk)
 
 	// Client: stores Server's public key.
-	verifier := NewVerifier(pk, crypto.SHA384)
+	client, err := NewClient(SHA384PSSRandomized, pk)
+	if err != nil {
+		fmt.Printf("failed to invoke a client: %v", err)
+		return
+	}
 
 	// Protocol (online)
 
-	// Client blinds a message.
+	// Client prepares the message to be signed.
 	msg := []byte("alice and bob")
-	blindedMsg, state, err := verifier.Blind(rand.Reader, msg)
+	preparedMessage, err := client.Prepare(rand.Reader, msg)
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "client failed to generate blinded message: %v", err)
+		fmt.Printf("client failed to prepare the message: %v", err)
+		return
+	}
+
+	// Client blinds a message.
+	blindedMsg, state, err := client.Blind(rand.Reader, preparedMessage)
+	if err != nil {
+		fmt.Printf("client failed to generate blinded message: %v", err)
 		return
 	}
 
 	// Server signs a blinded message, and produces a blinded signature.
 	blindedSignature, err := server.BlindSign(blindedMsg)
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "server failed to sign: %v", err)
+		fmt.Printf("server failed to sign: %v", err)
```

Comment:
why move the error from stderr to stdout?

Reply:
Because the Go tester program will check the output in stdout:
```go
// Output: Valid signature: true
```